apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/ConfigUtils.java

```diff
@@ -0,0 +1,48 @@
+package gov.cms.bfd.pipeline.rda.grpc;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This helper class abstracts the source of configuration values and handles insertion of default
+ * values when none are provided by the environment. Currently implemented to pull values from
+ * environment variables.
+ */
+class ConfigUtils {
```

Comment:
Yes I agree this would be very useful at a higher layer but I was reticent about making changes in the main code base at this point.   AppConfiguration is going to wind up loading these configs at some point but not sure it's the best place right now.  I was holding off until the orchestration PR has been merged.